src/config/config.go

```diff
@@ -66,6 +71,12 @@ type BtcRPC struct {
 	Cert   string `mapstructure:"cert"`
 }
 
+// EthRPC config for ethrpc
+type EthRPC struct {
+	Server string `mapstructure:"server"`
+	Port   string `mapstructure:"port"`
```

Comment:
Validate that this node is reachable in `Config.Validate()`, same as how btc node availability is checked there